tests/fixtures/isolation.py

```
@@ -0,0 +1,536 @@
+                task.cancel()
+
+            # Give tasks a chance to handle cancellation
+            if pending:
+                new_loop.run_until_complete(
+                    asyncio.gather(*pending, return_exceptions=True)
+                )
+
+            new_loop.close()
+        except Exception:
+            # Best effort cleanup
+            pass
```

Ruff (0.14.6)

`try`-`except`-`pass` detected, consider logging the exception

(S110)